src/content/docs/apm/agents/ruby-agent/background-jobs/sidekiq-instrumentation.mdx

```diff
@@ -27,6 +27,41 @@ By default this feature is off in case your job arguments contain sensitive info
   This feature is distinct from the generic [`capture_params`](/docs/agents/ruby-agent/installation-configuration/ruby-agent-configuration#capture_params) top-level setting, which controls whether HTTP request parameters are captured on transaction traces and traced errors for web requests. You can configure these two settings independently.
 </Callout>
 
+## Include or exclude certain job arguments [#include_or_exclude_arguments]
+
+Starting with Ruby agent version 9.5.0, an instance of the agent configured to capture Sidekiq job arguments as described above can additionally be configured to include or exclude certain arguments for fine grained control over which arguments are reported to New Relic. The `:'sidekiq.args.include'` configuration option can be set to an array of strings that will collectively serve as an allowlist. Each of those strings will be passed to `Regexp.new` for full regular expression support. For job arguments that are hashes, if a hash's key matches one of the include patterns, then both the key and its corresponding value will be included. For scalar arguments, the string representation of the scalar will need to match one of the include patterns to be captured. The `:'sidekiq.args.exclude'` option works similarly. It can be set to an array of strings that will each be passed to `Regexp.new` to create patterns. These patterns will collectively serve as a denylist for unwanted job args. Any hash key, hash value, or scalar that matches an exclude pattern will be excluded (not sent to New Relic).
+
+** newrelic_rpm 9.5.0 or higher**: configure `sidekiq.args.include` and `sidekiq.args.exclude`
+
+`newrelic.yml` examples:
+
+Any string in the `:'sidekiq.args.include'` or `:'sidekiq.args.exclude'` arrays will be turned into a regular expression. Knowledge of [Ruby regular expression support](https://ruby-doc.org/3.2.2/Regexp.html) can be leveraged but is not required. If regular expression syntax is not used, inexact matches will be performed and the string "Fortune" will match both "Fortune 500" and "Fortune and Glory". For exact matches, use [regular expression anchors](https://ruby-doc.org/3.2.2/Regexp.html#class-Regexp-label-Anchors).
```

Comment:
I like these links, but I'm somewhat concerned about their longevity, given that they're tied to a specific Ruby version. I don't know if there's a great alternative given how the Ruby Docs site is organized.

Reply:
(As discussed offline) @kaylareopelle and I will continue to look at alternatives for sharing Ruby specific Regexp documentation with readers. And we'll use a follow-up PR if we think of anything.

For now...

- By going with the latest stable Ruby version, we're hoping for at least several years of longevity in the links
- Hopefully readers familiar with the RubyDocs site will see the `3.2.2/Regexp.html` path and know that they can change the '3.2.2' bit and/or perform a web search for "Ruby Regexp" if the link expires
- The bit about Regexp anchors - though the link specifies Ruby v3.2.2 - fortunately hasn't changed since Ruby 1 and isn't anticipated to change, so any "Ruby Regexp anchors" doc should apply.